### PR TITLE
When used with data manager, column picker should properly show displayed columns

### PR DESCRIPTION
**Pull request by Blackbaud-AlexKingman**

### CODE REVIEW

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-ag-grid?_sp.spa=pr-col-picker-fix](https://developer.blackbaud.com/skyux-ag-grid?_sp.spa=pr-col-picker-fix)

**Review by johnhwhite — APPROVED** [review]

File: src/app/public/modules/ag-grid/ag-grid-data-manager-adapter.directive.ts [hunk 1]
```diff
@@ -142,6 +142,7 @@ export class SkyAgGridDataManagerAdapterDirective implements AfterContentInit, O
     });
   }
 
+  // TODO fix the displayed columns in here!
```
Comment (Blackbaud-TrevorBurch) [inline src/app/public/modules/ag-grid/ag-grid-data-manager-adapter.directive.ts:145]: Can you explain what this means?

File: src/app/public/modules/ag-grid/ag-grid-data-manager-adapter.directive.ts [hunk 2]
```diff
@@ -171,7 +172,7 @@ export class SkyAgGridDataManagerAdapterDirective implements AfterContentInit, O
       .pipe(takeUntil(this.ngUnsubscribe))
       .subscribe((event: ColumnMovedEvent) => {
         let columnOrder = agGrid.columnApi.getAllDisplayedVirtualColumns().map(
-          col => col.getColDef().colId
+          col => col.getColDef().colId || col.getColDef().field
```
Comment (Blackbaud-TrevorBurch) [inline src/app/public/modules/ag-grid/ag-grid-data-manager-adapter.directive.ts:175]: Ugh..... this gives me heartburn..... but I don't know of a better way to do this......

File: src/app/visual/data-manager/data-view-grid.component.ts [hunk 3]
```diff
@@ -49,13 +48,11 @@ export class DataViewGridComponent implements OnInit {
       lockVisible: true
     },
     {
-      colId: 'name',
       field: 'name',
       headerName: 'Fruit name',
       width: 150
     },
     {
-      colId: 'description',
```
Comment (Blackbaud-TrevorBurch) [inline src/app/visual/data-manager/data-view-grid.component.ts:58]: Should we leave this in one column to test a mixed case scenario?

**Review by Blackbaud-TrevorBurch — COMMENTED** [review]
@Blackbaud-AlexKingman - Two questions and one comment that you can ignore 😆

File: src/app/public/modules/ag-grid/ag-grid-data-manager-adapter.directive.ts (hunk 1 above)
Comment (Blackbaud-AlexKingman) [reply src/app/public/modules/ag-grid/ag-grid-data-manager-adapter.directive.ts:145]: That was an accidental commit. Removed.

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-ag-grid?_sp.spa=pr-col-picker-fix](https://developer.blackbaud.com/skyux-ag-grid?_sp.spa=pr-col-picker-fix)

**Comment by Blackbaud-AlexKingman** [comment]
Updating our requirements to require `colId`. Closing.

**Pull request closed without merging**